pandas/core/generic.py

```diff
@@ -6835,6 +6840,78 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Returns
         -------
         shifted : %(klass)s
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'Col1': [10, 20, 30, 20, 15, 30, 45],
```

Comment:
Not a bad example but I think we could save some space - could you limit the example to 5 rows of data? Shouldn't drastically affect your examples below